_docker/drupal/drupal-filesystem/web/config/sync/core.entity_view_display.node.learning_path.default.yml

```diff
@@ -36,14 +43,14 @@ content:
     region: content
   field_audience:
     weight: 102
-    label: above
+    label: visually_hidden
```

Comment:
We want to make this hidden, not visually hidden. I believe Luke has a ticket to change all of these in all content types.